src/components/TokenEnterAmount.test.tsx

```diff
@@ -215,18 +215,18 @@ describe('TokenEnterAmount', () => {
       await act(async () => {
         result.current.handleToggleAmountType()
       })
-      await act(async () => result.current.handleAmountInputChange('1234.67'))
-      await act(async () => result.current.handleAmountInputChange('1234.678'))
+      await act(async () => result.current.handleAmountInputChange('Bs1234.67'))
+      await act(async () => result.current.handleAmountInputChange('Bs1234.678'))
```

Comment:
this is the key change in the tests - the local currency symbol does come through this function at runtime, so it should be included here